tests/Agent/UnitTests/Core.UnitTest/Spans/SpanEventMakerTests.cs

```diff
@@ -377,6 +375,37 @@ public void GetSpanEvent_DoesNotIncludesErrorAttributes_WhenThereIsAnError_IfErr
             );
         }
 
+        [TestCase(true)]
+        [TestCase(true)]
```

Comment:
I think that you intended one of these `TestCase` attributes to be `TestCase(false)`.

Reply:
copy-paste nightmare